### ISSUE TITLE
Hello, Can you give an example of how to use it ?

### ISSUE THREAD

**Issue by FannyChung** [AnilSener/semiKmeans#1]
when use `fit_transform(self,texts,labels):` ,what are texts and labels form? when we use `predict` ,is X a vector?
Thank you !~
